py/torch_tensorrt/dynamo/conversion/impl/normalization/ops.py

```diff
@@ -152,16 +170,22 @@ def layer_norm_no_plugin(
     name: str,
     input: TRTTensor,
     normalized_shape: List[int],
-    weight: torch.Tensor,
-    bias: torch.Tensor,
-    eps: List[float],
+    weight: Optional[Union[TRTTensor, torch.Tensor, np.ndarray]],
+    bias: Optional[Union[TRTTensor, torch.Tensor, np.ndarray]],
+    eps: float,
 ) -> Union[TRTTensor, Sequence[TRTTensor]]:
     if not isinstance(input, TRTTensor):
         raise RuntimeError(
             f"LayerNorm received input {input} that is not part "
             "of the TensorRT region!"
         )
 
+    if weight is None:
+        weight = np.array(1.0)
+
+    if bias is None:
+        bias = np.array(0.0)
```

Comment:
See above comment

Reply:
Since line 189 is `shape = weight.shape` and lines 191 and 192 call `weight.reshape` and `bias.reshape`, I think `weight` and `bias` shouldn't be scalars.

Reply:
I see - in that case, it might be preferable to use `to_numpy(0.0)`, for instance, to get back a default-formatted numpy array for the float default. Additionally, I noticed the code below has some issues:

```python
gamma = to_numpy(weight.reshape(*shape))
##### Above is invalid, since the reshape should apply to the numpy output. It should instead be:
gamma = to_numpy(weight).reshape(shape)
```
The same as the above applies for `beta`.

Additionally, lines 194 - 196 should be using `get_axes_for_reduce_op`, as here:
https://github.com/pytorch/TensorRT/blob/ecdc040aec3902cbc396648043f54c96e6a91ee7/py/torch_tensorrt/dynamo/conversion/converter_utils.py#L209